apps/condo/domains/meter/utils/serverSchema/index.js

```diff
@@ -33,11 +35,12 @@ const getAvailableResidentMeters = async (context, userId) => {
         const residentPropertyId = get(resident, ['property', 'id'])
         const residentUnitName = get(resident, 'unitName')
 
-        const serviceConsumers = await ServiceConsumer.getAll(context, {
+        const serviceConsumers = await find('ServiceConsumer', {
             resident: { id: resident.id, deletedAt: null },
```

Comment:
why consumers for each resident is unique query? Why not something like `{id_in: residentsIds}`